### ISSUE TITLE
Issue with Cars196

### ISSUE THREAD

**Issue by riccardocl9** [aimagelab/mammoth#69]
Hi,

I would like to test several methods on `seq_cars196` (to replicate the results in "Semantic Residual Prompts for Continual Learning"), but automatic download is broken, and I don't find any guidelines to download the dataset in a format compatible with the class `MyCars196`. Can you provide any tip for solving this issue?

Off topic: https://github.com/aimagelab/mammoth/blob/659a3a8d85e5d032986b1e0c87eae116dd3627a2/models/coda_prompt.py#L46

Here, the last parameter is set to 0, but it means that the orthogonality loss in CODA is ignored. The original code mulitplies the orthogonality penalty by this parameter while you scale the loss twice: as in the original code in `CodaPrompt(nn.Module).forward()` (and since you set it to 0, the loss returned is 0) and in `CodaPrompt(ContinualModel).observe()` (but the loss is 0 and therefore $\mu\neq 0$ is ignored).
